examples/shader/post_processing.rs

```diff
@@ -0,0 +1,340 @@
+//! Shows how to render to a texture. Useful for mirrors, UI, or exporting images.
+
+use bevy::{
+    core_pipeline::{
+        draw_3d_graph, node, AlphaMask3d, Opaque3d, RenderTargetClearColors, Transparent3d,
+    },
+    ecs::system::{lifetimeless::SRes, SystemParamItem},
+    prelude::*,
+    reflect::TypeUuid,
+    render::{
+        camera::{ActiveCamera, Camera, CameraTypePlugin, RenderTarget},
+        render_asset::{PrepareAssetError, RenderAsset, RenderAssets},
+        render_graph::{Node, NodeRunError, RenderGraph, RenderGraphContext, SlotValue},
+        render_phase::RenderPhase,
+        render_resource::{
+            BindGroup, BindGroupDescriptor, BindGroupEntry, BindGroupLayout,
+            BindGroupLayoutDescriptor, BindGroupLayoutEntry, BindingResource, BindingType,
+            Extent3d, SamplerBindingType, ShaderStages, TextureDescriptor, TextureDimension,
+            TextureFormat, TextureSampleType, TextureUsages, TextureViewDimension,
+        },
+        renderer::{RenderContext, RenderDevice},
+        view::RenderLayers,
+        RenderApp, RenderStage,
+    },
+    sprite::{Material2d, Material2dPipeline, Material2dPlugin, MaterialMesh2dBundle},
+};
+
+#[derive(Component, Default)]
+pub struct FirstPassCamera;
+
+// The name of the final node of the first pass.
+pub const FIRST_PASS_DRIVER: &str = "first_pass_driver";
+
+fn main() {
+    let mut app = App::new();
+    app.insert_resource(Msaa { samples: 4 }) // Use 4x MSAA
```

Comment:
4 is the default, no need to insert the `Msaa` resource

Reply:
I guess we could clean those out then (probably in another PR) 
![image](https://user-images.githubusercontent.com/2290685/169259526-a3e6391c-809f-44ec-abcd-dafda8ebd445.png)

Reply:
Yes please.